docs/source/analysis.ipynb

```diff
@@ -0,0 +1,530 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction\n",
+    "\n",
+    "**Welcome to Keypoint MoSeq Analysis Visualization Notebook!**\n",
+    "\n",
+    "This notebook contains a collection of analyses we typically perform on behavioral recordings to better understand mouse behavior. Examples of such analyses are:\n",
+    "- adding experimental group labels to the session\n",
+    "- descriptive labeling of individual syllables\n",
+    "- comparing syllable usage across different experimental groups\n",
+    "- comparing syllable transition structure across different experimental groups\n",
+    "\n",
+    "The goal of this notebook is to provide you with a basic summary of syllable usage and sequencing statistics across experimental conditions and to help generate more specific questions related to observations you may identify here.\n",
+    "If you want to perform more specific analyses, you can export the MoSeq data (see below) to use with the tool (i.e., Excel, Python, R, Matlab) you're most comfortable with.\n",
+    "\n",
+    "In this notebook, the **Markdown** above each cell describes the purpose of the cell(s), cell output, and the instructions for running the cell(s) and/or interacting with the widget. The **inline code comments** in the code block provides contextual information about the function, code structure, and parameters.\n",
+    "\n",
+    "***"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Project setup\n",
+    "\n",
+    "## Files and Directory Structure\n",
```

Comment:
@calebweinreb  Overal proposal: all they should need at this point is a "project_dir" and a "name"
I think it is better to let people know that they should have the movies and trajectory plots etc at this stage too otherwise the visualization component won't work

Reply:
Copying @calebweinreb comments over
- I think we shuldn't assume the user already has `progress.yaml`
- If index.yaml is created at a later point in the notebook, we shouldn't include it in this list, since you're saying "you need the following files and folders" and it's not yet needed.

Reply:
addressed here: f368620a4a4f3f2f09caf399b213279beeed5a83